packages/theme/assets/styles/components/_forms.scss

```diff
@@ -68,6 +68,52 @@ form{
       top:0;
     }
   }
+
+  &__checkbox{
+    input[type="checkbox"]{
+      @include set-webaim-hidden();
+      &:checked{
+        + .form__checkbox-label{
+          &:before{
+            background: $color-brand-1;
+          }
+          &:after{
+            transform:scale(1);
+          }
+        }
+      }
+      &:focus + .form__checkbox-label:before{
+        box-shadow: inset 0 0 0 1px white, 0 0 3px 1px $color-brand-1;
+      }
+    }
+  }
+  &__checkbox-label{
+    &:before{
+      content:'';
+      height: $gutter-width;
+      width: $gutter-width;
+      background: $color-lt;
+      border:1px solid $color-brand-2;
+      position:absolute;
+      left:0;
+      top:0;
+      transition:background 0.2s ease-in-out;
+      cursor:pointer;
+    }
+    &:after{
+      content: "\2713";
```

Comment:
Because content can be read by screen readers I recommend anytime you are adding an icon this way to do so on an element with aria-hidden="true" I have made this change in another commit.